Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino

```diff
@@ -485,6 +493,35 @@ void set_chann(){
   }
 }
 
+void set_inv_iq(){
```

Comment:
good implementation, no problem, maybe i would add the use case in the help menu